remake_figures.yml

```diff
@@ -18,28 +18,13 @@ sources:
   - R/wqp_mapping_functions.R
   - R/readWQPdataPaged.R
   - R/get_all_us_secchi.R
-  - R/FIGURE_all_us_secchi.R
+  - R/FIGURE_all_us_secchi.R 
```

Comment:
I will get rid of that one. I just use remake for the fig generation because the processing times on certain steps are long